### ISSUE TITLE
RangeSelector does not pass all jest-axe accessibility checks

### ISSUE THREAD

**Issue by halocline** [grommet/grommet#5500]
### Expected Behavior

RangeSelector should not have accessibility violations and pass this test:
```
test('should not have accessibility violations', async () => {
    const { container } = render(
      <Grommet>
        <RangeSelector values={[20, 30]} />
      </Grommet>,
    );

    const results = await axe(container);
    expect(results).toHaveNoViolations();

    expect(container.firstChild).toMatchSnapshot();
  });
```

### Actual Behavior

Running latest `jest-axe` (`^5.0.0`) which relies on `axe-core` (`^4.2.0` rule set) catches the following accessibility error:
![Screen Shot 2021-08-02 at 1 30 45 PM](https://user-images.githubusercontent.com/1756948/127913566-304c740e-ce90-4c60-bcfb-642ad8fb58a2.png)

**Background**

- Current RangeSelector places an `aria-label="Lower Bounds"` and `aria-label="Upper Bounds"`on the corresponding thumb's `div`.
- According to [Dequeue](https://dequeuniversity.com/rules/axe/4.2/aria-allowed-attr?application=axeAPI) and [W3.org](https://www.w3.org/TR/html-aria/#dfn-no-corresponding-role), the use of any `aria-*` attribute on a `div` should be accompanied by a `role` attribute to better communicate with assistive technologies.
- `role="slider"` may be the most appropriate (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role)
  - If using `role="slider"`, also note advisement for use of `aria-valuemin`, `aria-valuemax`, and `aria-valuenow`.

### Steps to Reproduce

- Modify `package.json` to use `"jest-axe": "^5.0.1"`
- `yarn install`
- `yarn test`

### Your Environment

- Grommet version: 2.17.4
- Browser Name and version:
- Operating System and version (desktop or mobile):
